lib/brook/server.ex

```diff
@@ -38,6 +40,24 @@ defmodule Brook.Server do
   end
 
   def handle_call({:process, %Brook.Event{} = event}, _from, state) do
+    process(event, state)
+    {:reply, :ok, state}
+  end
+
+  def handle_call({:send, type, author, event}, _from, state) do
+    :ok = apply(state.driver.module, :send_event, [type, author, event])
+    {:reply, :ok, state}
+  end
+
+  def handle_cast({:process, %Brook.Event{} = event}, state) do
+    process(event, state)
+    {:noreply, state}
+  end
+
+  defp process(%{forwarded: false} = event, state) do
```

Comment:
I worry it could get confusing when you have a `process/1` as part of the `Brook.Event` api and a private `process/2` function here.